egs2/chime7_task1/asr1/run.sh

```diff
@@ -142,34 +141,32 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
     local/run_gss.sh --manifests-dir $manifests_root --dset-name $dset_name \
           --dset-part $dset_part \
           --exp-dir $gss_dump_root \
-          --cmd $cmd_gss \
+          --cmd "$cmd_gss" \
           --nj $ngpu \
           --max-segment-length $max_segment_length \
           --max-batch-duration $gss_max_batch_dur \
           --channels $channels \
-          --use-selection $use_selection
+          --use-selection $use_selection \
+          --top-k $top_k
     log "Guided Source Separation processing for ${dset_name}/${dset_part} was successful !"
   done
 fi
 
 
 if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
-
-  asr_train_set=kaldi/train_all_mdm_ihm_rvb_gss
-  asr_cv_set=kaldi/chime6/dev/gss # use chime only for validation
   # Decoding on dev set because test is blind for now
-  # NOTE that ESPNet will not make copies of the original Kaldi manifests
-  # e.g. for training and cv, so if you set $train_max_segment_length these
-  # will be discarded also from the test set (if the test set is the same as evaluation)
-  # you need to make a copy !
-  ./utils/copy_data_dir.sh data/kaldi/chime6/dev/gss data/kaldi/chime6/dev/gss_inf
-  asr_tt_set="kaldi/chime6/dev/gss_inf kaldi/dipco/dev/gss/ kaldi/mixer6/dev/gss/"
 
   pretrained_affix=
   if [ -n "$use_pretrained" ]; then
-    asr_train_set=kaldi/dipco/dev/gss # dummy one, it is not used
+    asr_train_set="" #kaldi/dev_ihm_all # dummy one, it is not used
+    asr_cv_set=""  #kaldi/dev_ihm_all
     pretrained_affix+="--skip_data_prep false --skip_train true "
     pretrained_affix+="--download_model ${use_pretrained}"
+  else
+    asr_train_set=kaldi/train_all_mdm_ihm_rvb_gss
+    asr_cv_set=kaldi/chime6/dev/gss # use chime only for validation
+    # note that if it is also in test a copy named org/${valid} is created
+    # will have to handle it in stage 4
   fi
 
```

Comment:
```suggestion
    pretrained_affix+="--skip_data_prep false --skip_train true "
    pretrained_affix+="--download_model ${use_pretrained}"
  else
    # note that if it is also in test a copy named org/${valid} is created
    # will have to handle it in stage 4
  fi
  asr_train_set=kaldi/train_all_mdm_ihm_rvb_gss
  asr_cv_set=kaldi/chime6/dev/gss # use chime only for validation
```

You don't need to change asr_train_set and asr_cv_set.  You must keep `asr_cv_set=kaldi/chime6/dev/gss` to evaluate the original set.

Reply:
The behavior when `--skip_train true` in this case is as following:

### Stage 3
- `data/org/${train_set}`: skipped
- `data/org/${valid_set}`: Generated
- `data/${test_set}` for all test_sets excluding valid_set: Generated

### Stage 4: skipped
### Decoding
- `data/org/${valid_set}` and `data/${test_set}`  for all test_sets excluding valid_set are evaluated

Reply:
You are right. 

What about ? 
```bash
  if [ -n "$use_pretrained" ]; then
    asr_train_set="dummy_tr" # dummy one, it is not used
    asr_cv_set="dummy_cv"
    pretrained_affix+="--skip_data_prep false --skip_train true "
    pretrained_affix+="--download_model ${use_pretrained}"
  else
    asr_train_set=kaldi/train_all_mdm_ihm_rvb_gss
    asr_cv_set=kaldi/chime6/dev/gss # use chime only for validation
    # we will make a copy to avoid ESPNet modify this 
    ./utils/copy_data_dir.sh ./data/$asr_cv_set ./data/${asr_cv_set}_copy
    asr_cv_set=${asr_cv_set}_copy
  fi
```

Reply:
Please fix this issue, then I'll merge

Reply:
Ok I will do in this way for now I think.

Reply:
Sorry, i missed your comment by some delay.

```
    asr_train_set="dummy_tr" # dummy one, it is not used
    asr_cv_set="dummy_cv"
```

can't solve the problem. Please follow my fix.

Reply:
Could you fix this issue?